xml/System.Security.Cryptography/ECDsaOpenSsl.xml

```diff
@@ -37,8 +45,20 @@
       </AssemblyInfo>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.ECDsaOpenSsl" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+This constructor does not generate a new public/private keypair immediately.
+This constructor sets the <xref:System.Security.Cryptography.ECDsaOpenSsl.KeySize> property to 521 and when a key is needed the saved size is used to identify the target curve.
+If a key is loaded via the <xref:System.Security.Cryptography.ECDsaOpenSsl.ImportParameters%2A> method, or other key import method, the key size from this constructor has no meaning.
+ 
+  
+ ]]></format>
+        </remarks>
+        <altmember cref="M:System.Security.Cryptography.ECDsa.Create"/>
```

Comment:
The build [had a warning](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C25%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C201911251733433084-3564%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CFGX0qjAqjq356%2FYCreWkHb53MV0l24SaaGm0YaZBmc%3D&st=2019-11-25T18%3A07%3A32Z&se=2019-12-26T18%3A12%3A32Z&sp=r) stating that this cref does not exist ([here's the preview of this constructor](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecdsaopenssl.-ctor?view=dotnet-plat-ext-3.0&branch=pr-en-us-3564#System_Security_Cryptography_ECDsaOpenSsl__ctor_System_Int32_)).

The ECDsa.xml at the top of this review shows that these are the available DocIds for the Create overloads. Which one did you mean to use?:

`M:System.Security.Cryptography.ECDsa.Create`
`M:System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECCurve)`
`M:System.Security.Cryptography.ECDsa.Create(System.Security.Cryptography.ECParameters)`
`M:System.Security.Cryptography.ECDsa.Create(System.String)`